xlators/mgmt/glusterd/src/glusterd-locks.c

```diff
@@ -497,7 +497,9 @@ glusterd_mgmt_v3_lock(const char *name, uuid_t uuid, uint32_t *op_errno,
     gf_boolean_t is_valid = _gf_true;
     uuid_t owner = {0};
     xlator_t *this = THIS;
+#ifdef DEBUG
     char *bt = NULL;
+#endif
```

Comment:
This could be moved into the following `#ifdef DEBUG` block to avoid too many conditional blocks.

Reply:
Ack